akka-cluster-sharding/src/main/scala/akka/cluster/sharding/ShardCoordinator.scala

```diff
@@ -743,26 +774,35 @@ abstract class ShardCoordinator(
 
       case RebalanceDone(shard, ok) =>
         rebalanceWorkers -= sender()
-        if (ok)
-          log.debug("Rebalance shard [{}] completed successfully.", shard)
-        else
-          log.warning("Rebalance shard [{}] didn't complete within [{}].", shard, handOffTimeout.pretty)
-        // The shard could have been removed by ShardRegionTerminated
-        if (state.shards.contains(shard)) {
-          if (ok) {
+
+        val logContext =
+          if (gracefulShutdownInProgress(state.shards(shard))) "shutdown"
+          else "rebalance"
```

Comment:
I dislike this and I see a potential confusion logging. We can have a general rebalance followed by a region shutdown. Then, we get the `RebalanceDone` and we log that the shard is dealloacted because of a shutdown. 

Not big issue I think. At the end, there is a shutdown happening anyway. So, we are not really lying. ;-)

Reply:
I would prefer to not have `logContext` and simply say: "Shard dealloacted"

WDYT?

Reply:
Ok, we can use "deallocated".